spec/system/user/user_updates_announcements_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe "Announcements", type: :system do
+  let(:user) { create(:user) }
+
+  before do
+    sign_in(user)
+    visit "/settings/misc"
+  end
+
+  it "renders the announcements settings" do
+    expect(page).to have_text("Announcements")
```

Comment:
What is this test actually checking for? It seems to do the same thing as the [one over here](https://github.com/thepracticaldev/dev.to/pull/8343/files#diff-dabdd41261904eae23b477c8ae3a61adR34-R37).